src/jshint.js

```diff
@@ -2218,6 +2217,19 @@ var JSHINT = (function() {
     return this;
   }));
   prefix("new", function() {
+    var mp = metaProperty("target");
```

Comment:
you already specify the name once, I wonder if it would be cleaner if metaProperty only returns if the value === "target" then you won't need to check it again below ?